tromega/lib/views/plans/filter_exercises.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:tromega/views/plans/add_exercise.dart';
+import 'package:tromega/views/plans/edit_training_view.dart';
+import 'package:tromega/views/in_training/track_training_view.dart';
+
+import '../../widgets/plan/exercise_container_plan.dart';
+import 'package:flutter/material.dart';
+import '../../widgets/bottom_menu.dart';
+import '../../widgets/shared/app_bar.dart';
+import 'package:tromega/data/trainingDay.dart';
+import 'package:tromega/data/filter.dart';
+import 'package:tromega/data/trainingPlan.dart';
+import 'package:multi_select_flutter/multi_select_flutter.dart';
+
+class FilterView extends StatefulWidget {
+  //Visualisierung von einem Tag eines Trainingsplans
+  FilterView({Key? key, required this.filter}) : super(key: key);
+  Filter filter;
+
+  @override
+  State<FilterView> createState() => _FilterViewState();
+}
+
+class _FilterViewState extends State<FilterView> {
+  List<String> _selectedMuscle = [];
+  List<String> _selectedEquipment = [];

```

Comment:
Die beiden vielleicht in onInit() erst initialisieren?

Reply:
Sehe gerade, wurde in intiState() sowieso überschrieben, dann hier vielleicht nur deklarieren und nicht initialisieren